src/views/Completed.vue

```diff
@@ -3,192 +3,212 @@
+            <ion-checkbox slot="start"/>
+            <ion-label>
+              {{ carrierPartyId.val }}
+              <p>{{ carrierPartyId.groups }} {{ carrierPartyId.groups === 1 ? $t('package') : $t("packages") }}</p>
+            </ion-label>
+            <ion-icon :icon="printOutline" />
+          </ion-item>
 
-          <div class="order-metadata">
+          <ion-item lines="none" v-for="shipmentMethod in shipmentMethods" :key="shipmentMethod.val">
+            <ion-checkbox :value="shipmentMethod.val" slot="start"/>
             <ion-label>
-              Next Day Shipping
-              <p>{{ $t("Ordered") }} 28th January 2020 2:32 PM EST</p>
+              {{ shipmentMethod.val }}
+              <p>{{ shipmentMethod.groups }} {{ shipmentMethod.groups > 1 ? $t('orders') : $t('order') }}, {{ shipmentMethod.itemCount }} {{ shipmentMethod.itemCount > 1 ? $t('items') : $t('item') }}</p>
             </ion-label>
-          </div>
+          </ion-item>
         </div>
 
-        <div class="order-item">
-          <div class="product-info">
-            <ion-item lines="none">
-              <ion-thumbnail slot="start">
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
-              </ion-thumbnail>
+        <ion-button expand="block" class="bulk-action desktop-only" fill="outline" @click="shipOrderAlert">{{ $t("Ship") }}</ion-button>
+
+        <ion-card v-for="(orders, index) in completedOrders.list" :key="index">
+          <div class="card-header">
+            <div class="order-primary-info">
               <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
+                {{ orders.doclist.docs[0].customerName }}
+                <p>{{ $t("Ordered") }} {{ formatUtcDate(orders.doclist.docs[0].orderDate, 'dd MMMM yyyy t a ZZZZ') }}</p>
               </ion-label>
-            </ion-item>
-          </div>
+            </div>
 
-          <div class="product-metadata mobile-only">
-            <ion-note>49 {{ $t("pieces in stock") }}</ion-note>
-          </div>
-        </div>
+            <div class="order-tags">
+              <ion-chip outline>
+                <ion-icon :icon="pricetagOutline" />
+                <ion-label>{{ orders.doclist.docs[0].orderId }}</ion-label>
+              </ion-chip>
+            </div>
 
-        <div class="order-item">
-          <div class="product-info">
-            <ion-item lines="none">
-              <ion-thumbnail slot="start">
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
-              </ion-thumbnail>
+            <div class="order-metadata">
               <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
+                {{ orders.doclist.docs[0].shipmentMethodTypeDesc }}
```

Comment:
We will not be accessing data with doclist properties, we should transform and use order & items. As discussed to be taken care in upcoming PR